### PR TITLE
gh-1960 Use chain-related execution parameters

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Handles #1960 

Changes proposed in this pull request:
- Use only the 'message call' related values received from the web connection request

### CODE REVIEW

File: Multisig/Features/Connect to Web/Logic/WebConnectionController.swift [hunk 1]
```diff
@@ -655,6 +658,40 @@ class WebConnectionController: ServerDelegateV2, RequestHandler, WebConnectionSu
...
+            let features = chain.features
+        else {
+            return
+        }
+
+        let minerTip: Sol.UInt256 = 1_500_000_000
+
+        // select tx type based on the chain.
+        let isEIP1559 = features.contains("EIP1559")
+        if isEIP1559 {
```
Comment (elgatovital) [inline Multisig/Features/Connect to Web/Logic/WebConnectionController.swift:675]: what about EIP2930?

File: Multisig/Features/Connect to Web/Logic/WebConnectionController.swift [hunk 2]
```diff
@@ -655,6 +658,40 @@ class WebConnectionController: ServerDelegateV2, RequestHandler, WebConnectionSu
...
+            request.transaction = Eth.TransactionEip1559(
+                chainId: Sol.UInt256(chainId),
+                from: request.transaction.from,
+                to: request.transaction.to,
+                value: request.transaction.value,
+                input: request.transaction.data,
+                fee: .init(maxPriorityFee: minerTip)
+            )
+        } else {
+            request.transaction = Eth.TransactionLegacy(
```
Comment (elgatovital) [inline Multisig/Features/Connect to Web/Logic/WebConnectionController.swift:685]: would EIP-2930 type be handled here correctly?

Comment (biafra23) [reply Multisig/Features/Connect to Web/Logic/WebConnectionController.swift:685]: Do we need to handle EIP-2930? We're not using access lists. Do we allow the creation of a tx with EIP-2930 specific featueres? For example access lists?

Comment (biafra23) [reply Multisig/Features/Connect to Web/Logic/WebConnectionController.swift:685]: If this handling is a problem, we need to change it in TransactionExecutionController.swift:L324 as well.

**Pull request merged**